src/contexts/notes-context.js

```diff
@@ -0,0 +1,89 @@
+import {
+	createContext,
+	useContext,
+	useEffect,
+	useReducer,
+	useState,
+} from "react";
+import { initialNotesState, notesReducerFunction } from "reducers/";
+import { useAuth } from "./";
+import { getNotesService } from "services/";
+import { useToastify } from "custom-hook/useToastify";
+
+const NotesContext = createContext(initialNotesState);
+const { Provider } = NotesContext;
+
+const NotesProvider = ({ children }) => {
+	const { isAuth } = useAuth();
+
+    const { showToast } = useToastify();
+
+	const fetchNotes = async (authToken) => {
+		try {
+			const {
+				data: { notes },
+			} = await getNotesService(authToken);
+			notesDispatch({
+				action: {
+					type: "SET_NOTES_SUCCESS",
+					payload: {
+						notes,
+						notesLoading: false,
+						notesError: null,
+						showNewNoteForm: false,
+						isEditing: false,
+						editingNoteId: -1,
+					},
+				},
+			});
+		} catch (error) {
+			notesDispatch({
+				action: {
+					type: "SET_NOTES_ERROR",
+					payload: {
+                        showNewNoteForm: false,
```

Comment:
you can use preetier that will do all this formatting

Reply:
Again not sure why this is happening! But I ran prettier and seems fine on VS code.

Reply:
I shall look into this.